.github/workflows/lint-workflow-files.yml

```
@@ -0,0 +1,35 @@
+name: "Lint GitHub Actions workflows"
+
+on:
+  pull_request:
+    paths:
+      - ".github/**"
+
+jobs:
+  actionlint:
+    runs-on: "ubuntu-24.04"
```

```suggestion
    runs-on: "ubuntu-latest"
```
I prefer the `latest` tag so we stay up-to-date without maintaining this file